client/verta/verta/runtime.py

```diff
@@ -192,15 +197,13 @@ class context:
         import json
         from verta import runtime
 
-        with runtime.context() as ctx:
-            runtime.log("key", {"value": ...})
+        with runtime.context(validate=True) as ctx:  # validate values are JSON-serializable
+            my_model.predict(x)  # your model, with your calls to runtime.log()
             print(json.dumps(ctx.logs()))
 
-        # After exiting the context manager:
+        # Same output after exiting the context manager:
         print(json.dumps(ctx.logs()))
 
-        # Output (both) = '{"key": {"value": ...}}'
-
```

Comment:
Open to feedback on this: I wanted the example to show `model.predict()`, since a user is much more likely to test that in the context than to call `runtime.log()` directly

Reply:
Makes sense to me.  The inline comments add some clarity as well.  Thumbs up